cmd/karavictl/cmd/storage_create.go

```diff
@@ -288,42 +287,38 @@ func NewStorageCreateCmd() *cobra.Command {
 						errAndExit(err)
 					}
 
-					var powermaxSymmetrix []*types.Symmetrix
-
+					// get all PowerMax system IDs
```

Comment:
line 290-294, with this changes we aren't looping thru each array in a unisphere, don't we have to delete those code?

Reply:
This call is ok because it is only retrieving a list of system IDs. The call that can have an issue if the something isn't right is `pmClient.GetSymmetrixByID`.